cmd/schema.go

```diff
@@ -0,0 +1,458 @@
+	cmd.MarkFlagRequired("host")
+
+	cmd.Flags().BoolVarP(&metadata, "metadata", "m", false, "set this flag to get metadata")
+	cmd.MarkFlagRequired("host")
+
+	cmd.Flags().StringVarP(&output, "output", "o", "", "path to the output file")
+
+	return cmd
+}
+
+func deleteSchemaCmd() *cobra.Command {
+	var host string
+	var req stencilv1beta1.DeleteSchemaRequest
+	var reqVer stencilv1beta1.DeleteVersionRequest
+	var version int32
+
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "delete all schemas",
+		Args:  cobra.ExactArgs(2),
+		Example: heredoc.Doc(`
+			$ stencil schema delete <namespace-id> <schema-id>
+	    `),
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			spinner := printer.Spin("")
+			defer spinner.Stop()
+
+			conn, err := grpc.Dial(host, grpc.WithInsecure())
+			if err != nil {
+				return err
+			}
+			defer conn.Close()
+
+			namespaceID := args[0]
+			schemaID := args[1]
+
+			client := stencilv1beta1.NewStencilServiceClient(conn)
+
+			if version == 0 {
+				req.NamespaceId = namespaceID
+				req.SchemaId = schemaID
+
+				_, err = client.DeleteSchema(context.Background(), &req)
+				if err != nil {
+					return err
+				}
+			} else {
+				reqVer.NamespaceId = namespaceID
+				reqVer.SchemaId = schemaID
+				reqVer.VersionId = version
+
+				_, err = client.DeleteVersion(context.Background(), &reqVer)
+				if err != nil {
+					return err
+				}
+
+			}
+
+			spinner.Stop()
+
+			fmt.Printf("schema successfully deleted")
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&host, "host", "", "stencil host address eg: localhost:8000")
+	cmd.MarkFlagRequired("host")
+
+	cmd.Flags().Int32VarP(&version, "version", "v", 0, "particular version to be deleted")
+
+	return cmd
+}
+
+func versionSchemaCmd() *cobra.Command {
+	var host string
+	var req stencilv1beta1.ListVersionsRequest
+
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "version(s) of all schemas",
+		Args:  cobra.ExactArgs(2),
+		Example: heredoc.Doc(`
+			$ stencil schema list <namespace-id> <schema-id>
+	    `),
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			spinner := printer.Spin("")
+			defer spinner.Stop()
+
+			conn, err := grpc.Dial(host, grpc.WithInsecure())
+			if err != nil {
+				return err
+			}
+			defer conn.Close()
+
+			namespaceID := args[0]
+			schemaID := args[1]
+
+			req.NamespaceId = namespaceID
+			req.SchemaId = schemaID
+
+			client := stencilv1beta1.NewStencilServiceClient(conn)
+			res, err := client.ListVersions(context.Background(), &req)
+			if err != nil {
+				return err
+			}
+
+			report := [][]string{}
+			versions := res.GetVersions()
+
+			spinner.Stop()
+
+			report = append(report, []string{"VERSIONS(s)"})
+
+			for _, v := range versions {
+				report = append(report, []string{
+					string(v),
+				})
```

Comment:
This is not printing versions, it won't convert int to string directly. consider using `strconv`